src/content/docs/index.mdx

```diff
@@ -116,43 +127,143 @@ Engine • revision e81fa131e59506d9f6af2a0cee7de749131f1bf0
+(back to 3.10.0). See [Flutter Version Management](/flutter-version) for more
+info.
+:::
+
+### Create a release
+
+We will create a release using the unmodified Counter app. Run:
+
+<Tabs>
+
+<TabItem value="android" label="Android">
+
+```sh
+shorebird release android
+```
+
+</TabItem>
+
+<TabItem value="ios" label="iOS">
 
+```sh
+shorebird release ios
 ```
-shorebird login
-The Shorebird CLI needs your authorization to manage apps, releases, and patches
-on your behalf.
 
-In a browser, visit this URL to log in:
+</TabItem>
+</Tabs>
 
-https://accounts.google.com/o/oauth2/v2/auth...
+When prompted, use the suggested version number (`1.0.0+1`), and enter `y` when
+asked if you would like to continue.
 
-Waiting for your authorization...
+### Preview the release
 
-🎉 Welcome to Shorebird! You are now logged in as <email>.
+To preview the release with Shorebird (that is, with [Shorebird's fork of the
+Flutter engine](/faq#how-does-shorebird-relate-to-flutter)), run:
 
-🔑 Credentials are stored in ./path/to/credentials.json.
-🚪 To logout use: "shorebird logout".
+```sh
+shorebird preview
 ```
 
-That's it 🎉
+Now kill the app on your device or emulator.
+
+### Create a patch
+
+We will now make a small change to the Counter app. In `lib/main.dart`, change
+the app theme's `primarySwatch` from blue to green:
+
+```diff
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        // This is the theme of your application.
+        //
+        // Try running your application with "flutter run". You'll see the
+        // application has a blue toolbar. Then, without quitting the app, try
+        // changing the primarySwatch below to Colors.green and then invoke
+        // "hot reload" (press "r" in the console where you ran "flutter run",
+        // or simply save your changes to "hot reload" in a Flutter IDE).
+        // Notice that the counter didn't reset back to zero; the application
+        // is not restarted.
+-       primarySwatch: Colors.blue,
++       primarySwatch: Colors.green,
+      ),
+      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+```
+
+After making this change, save the file and run:
+
+<Tabs>
 
-You now have a Shorebird account, have installed Shorebird CLI on your machine, and are ready to start using Shorebird.
+<TabItem value="android" label="Android">
+
+```sh
+shorebird patch android
+```
+
+</TabItem>
+
+<TabItem value="ios" label="iOS">
+
+```sh
+shorebird patch ios
+```
+
+</TabItem>
+
+</Tabs>
+
+### See the patch in action
+
+Launch the app from your device or emulator. The app will still have the
+original blue theme, but it will be downloading the patch we just created in the
+background. Kill and launch the app again, and the app will be green! 🎉
```

Comment:
Should we talk about what a user would expect to see log-wise in preview?

Reply:
Happy to do so, I was mainly just trying to consolidate what we already had before going in and workshopping the actual content haha.

Reply:
Makes total sense, we can do that in a later pass